docs/reference/modules/piece-page-type.md

```diff
@@ -142,13 +148,33 @@ module.exports = {
     ]
   }
 };
-
 ```
   <template v-slot:caption>
     modules/book-page/index.js
   </template>
 </AposCodeBlock>
 
+<AposCodeBlock>
+
+```
+{
+  _author: [
+    { value: 'Gibson', label: 'Gibson' },
```

Comment:
`_author` would be a relationship field, so the values are going to be IDs (as in `._id` properties of docs), not user-friendly strings.

Reply:
The labels will be friendly of course.

Reply:
Changed for output from a relationship field filter. I was going to change it to a simple `string` type, but I thought this would be a good example.